spec/amber/dsl/server_spec.cr

```diff
@@ -124,6 +124,38 @@ module Amber
           end
         end
       end
+
+      describe "routes" do
+        context "with a namespace" do
+          server = Amber::Server.instance
+          server.handler.pipeline.clear
+
+          Amber::Server.configure do
+            pipeline :web do
+            end
+
+            routes :web do
+              namespace "/test" do
+                get "/hello", HelloController, :index
+
+                namespace "/test2" do
+                  get "/hello", HelloController, :index
```

Comment:
Can you add a `resource` to a namespace?

Reply:
`resource` is in the same namespace as `get` `post` etc, I think so.

Reply:
@alex-lairan can you add a test for a resource inside the namespace?

Reply:
Yes, no problem !

Reply:
Done